docs/developer-docs/latest/setup-deployment-guides/typescript.md

```diff
@@ -48,7 +48,7 @@ yarn create-strapi-app@latest my-project --typescript
 
 ## Understand TypeScript support
 
-TypeScript-enabled Strapi applications have a `dist` directory at the project root and 2 `tsconfig.json` files (see [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md)).
+TypeScript-enabled Strapi applications have a `dist` directory at the project root, a `./dist/build/` subdirectory that contains the compiled admin panel, and 2 `tsconfig.json` files (see [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md)).
```

Comment:
I probably missed that with the first review, sorry 😅 :
I think we could simplify/update this sentence as it mostly repeats what's in the table. 
```suggestion
TypeScript-enabled Strapi applications have a specific [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md)) with the following dedicated folders and configuration files:
```

Reply:
agree.